### PR TITLE
added wishlist

### PR DESCRIPTION
**Pull request by Priyanshi-Tiwari**
127 0 0 1_5501_pages_wishlist html (2)

### CODE REVIEW

File: pages/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="../Stylesheets/main.css" />
+    <script
+      src="https://kit.fontawesome.com/6026902e47.js"
+      crossorigin="anonymous"
+    ></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Stellar Ecommerce Landing Page</title>
```
Comment (Sameony) [inline pages/wishlist.html]: Do try to give concise titles to the webpage

File: pages/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
```
Comment (vaibhav-mahalle) [inline pages/wishlist.html]: you can name it as Landing Page first and then your app name so page name will be more visible in the browser.

**Review by Sameony — APPROVED** [review]
Good clean code. Just a minor follow up regarding the title of the webpage

**Review by vaibhav-mahalle — APPROVED** [review]
suggested some minor changes please look into it.

File: pages/wishlist.html (hunk 1 above)
Comment (Priyanshi-Tiwari) [reply pages/wishlist.html]: Updated it

File: pages/wishlist.html (hunk 2 above)
Comment (Priyanshi-Tiwari) [reply pages/wishlist.html]: That's a good suggestion, updated it.

**Pull request merged**